Makefile

```diff
@@ -202,3 +202,9 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+.PHONY: generate-rules-spec
+generate-rules-spec:
```

Comment:
As mentioned below, I think it would be worth generating all of the files into the same go package, as this should allow us to produce better server stubs. BTW, in this case you can collapse these three generation calls into one single command, e.g.
```sh
@oapi-codegen -generate types,client,chi-server -package rules -o rules/rules.go
```
This means also that the Makefile target would not need to be phony.